apps/web/src/pages/integrations/IntegrationsListPage.tsx

```diff
@@ -63,10 +65,66 @@ const IntegrationsList = () => {
   const { integrations, loading: areIntegrationsLoading } = useIntegrations();
   const isLoading = areEnvironmentsLoading || areIntegrationsLoading;
   const hasIntegrations = integrations && integrations?.length > 0;
-  const data = useMemo<ITableIntegration[] | undefined>(
-    () => integrations?.map((el) => mapToTableIntegration(el, environments)),
-    [integrations, environments]
+
+  const {
+    data: { limit: emailLimit, count: emailCount },
+    loading: emailLimitLoading,
+  } = useIntegrationLimit(ChannelTypeEnum.EMAIL);
+  const {
+    data: { limit: smsLimit, count: smsCount },
+    loading: smsLimitLoading,
+  } = useIntegrationLimit(ChannelTypeEnum.SMS);
+
+  const noEmailIntegrations = useMemo(
+    () => integrations?.filter((el) => el.channel === ChannelTypeEnum.EMAIL).length === 0,
+    [integrations]
+  );
+  const noSmsIntegrations = useMemo(
+    () => integrations?.filter((el) => el.channel === ChannelTypeEnum.SMS).length === 0,
+    [integrations]
   );
+  const isNovuEmailActive = !emailLimitLoading && noEmailIntegrations && emailLimit - emailCount > 0;
+  const isNovuSmsActive = !smsLimitLoading && noSmsIntegrations && smsLimit - smsCount > 0;
+
+  const data = useMemo<ITableIntegration[] | undefined>(() => {
+    const mappedIntegrations = (integrations ?? []).map((el) => mapToTableIntegration(el, environments));
+    if (!IS_DOCKER_HOSTED) {
+      mappedIntegrations.unshift(
+        mapToTableIntegration({
+          _id: '-2',
+          _environmentId: '',
+          _organizationId: '',
+          name: 'Novu SMS',
+          identifier: '',
+          providerId: SmsProviderIdEnum.Novu,
+          channel: ChannelTypeEnum.SMS,
+          credentials: {},
+          active: isNovuSmsActive,
+          deleted: false,
+          deletedAt: '',
+          deletedBy: '',
+        })
+      );
+      mappedIntegrations.unshift(
+        mapToTableIntegration({
+          _id: '-1',
+          _environmentId: '',
+          _organizationId: '',
+          name: 'Novu Email',
+          identifier: '',
+          providerId: EmailProviderIdEnum.Novu,
+          channel: ChannelTypeEnum.EMAIL,
+          credentials: {},
+          active: isNovuEmailActive,
+          deleted: false,
+          deletedAt: '',
+          deletedBy: '',
+        })
+      );
+    }
```

Comment:
this is how we "imitate" the Novu integrations... these two integrations have "fake" `id` and no `identifier, environment, organization`... so we should be careful in the future code